### DISCUSSION TITLE
RoSocket Discussions

### DISCUSSION THREAD

**Discussion by richdentistboy** [RoSocket/rosocket#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by felixqx1** [comment]
I can't get it to send  proper JSON

**Comment by EngLenz** [comment]
Hey @richdentistboy, hope you’re doing well!

Thanks a lot for making RoSocket available. I really appreciate it. I’m currently trying to use it, but I’m running into some issues.

I’ve already set up a local WebSocket server, exposed it as WSS, and everything works correctly when testing via Postman — the server accepts connections and exchanges messages as expected.

However, when trying to connect from Roblox, it doesn’t connect. I also tested against a public WSS endpoint (wss://echo.websocket.org), and that doesn’t connect either from Roblox.

If you’re available, I’d really appreciate your help clarifying what might be missing or if there are any known limitations or requirements on the Roblox side.

Thanks in advance!
